youtubeBot/bot.py

```diff
@@ -34,6 +59,29 @@ def send_text(self, text):
 class YoutubeBot(Bot):
     pass  # TODO your code here!
 
+    # no need to define a constructor , default constructor will be generated .
+
+    # static variable -- Dictionary hold previous videos as pairs {message: video path}.
+    downloadedVideos = {}
```

Comment:
You should put this variable inside "__init__" method (class attribute)

Reply:
> You should put this variable inside "**init**" method (class attribute)

Yes correct , but i wanted  it to be a static variable to store downloaded videos , and this variable should not be linked with any instance of the class .